src/test/kotlin/no/nav/tilleggsstonader/sak/journalføring/JournalpostServiceTest.kt

```diff
@@ -114,4 +125,79 @@ class JournalpostServiceTest() {
             }.hasMessageContaining("Vedlegget er sannsynligvis under arbeid, må åpnes i gosys")
         }
     }
+
+    @Nested
+    inner class LogiskVedlegg {
```

Comment:
Burde det være en test på at det ikke skjer en oppdatering om det ikke er en endring?

Reply:
Det er det som verifiseres i den første testen for dete ene dokumentet

```kotlin
            verify(exactly = 0) {
                journalpostClient.oppdaterLogiskeVedlegg(
                    dokumentUtenLogiskVedlegg.dokumentInfoId,
                    any(),
                )
            }
```